core/lib/search/adapter/api/searchAdapter.ts

```diff
@@ -75,14 +76,36 @@ export class SearchAdapter {
     url = url + '/' + encodeURIComponent(Request.index) + '/' + encodeURIComponent(Request.type) + '/_search'
     url = url + '?' + queryString.stringify(httpQuery)
 
+    let queryMethod = config.elasticsearch.queryMethod
+    if (Request.type === 'product' && Request.sort) {
+      const sortType = Request.sort.split(':', 2)
```

Comment:
```suggestion
      const [ sortByField, sortOrder ] = Request.sort.split(':', 2)
```
Remember to update in different places